pyproject.toml

```diff
@@ -58,7 +63,7 @@ rred = "rred_reports.cli:main"
 minversion = "7.0"
 addopts = ["-ra", "--showlocals", "--strict-markers", "--strict-config"]
 xfail_strict = true
-filterwarnings = ["error"]
+filterwarnings = ["error", "ignore::DeprecationWarning"]
```

Comment:
oh interesting, what has a deprecation warning?

Reply:
ah that was `smtplib` which we're now not using, will get rid of this!